packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -1658,6 +1658,7 @@ Generated by [AVA](https://avajs.dev).
         LocalChainFacade_kindHandle: 'Alleged: kind',
         Orchestrator_kindHandle: 'Alleged: kind',
         RemoteChainFacade_kindHandle: 'Alleged: kind',
+        connectionToDetails: 'Alleged: weakMapStore',
```

Comment:
@turadg do we have sufficient fast-usdc upgrade test coverage to be confident about this change?